docs/features/font.md

```diff
@@ -252,6 +252,33 @@ ClassicEditor
 	.catch( ... );
 ```
 
+### Documents colors
+
+In addition to predefined list of colors, the plugin also includes document colors feature. It lists the colors used in the document and lists them in the order of occurrence. Duplicated colors are not inserted multiple times. By default number of document colors is limited, but you can adjust it (or disable it) using {@link module:font/fontcolor~FontColorConfig#documentColors `fontColor.documentColors`} or {@link module:font/fontbackgroundcolor~FontBackgroundColorConfig#documentColors `fontBackgroundColor.documentColors`} options.
```

Comment:
> In addition to predefined list of colors, the plugin also includes document colors feature.

@AnnaTomanek could you check this part? It doesn't read well.

Reply:
For me the plugin implements something and the UI includes something.

Reply:
FYI: I rewrite this part a little bit.